gflownet.py

```diff
@@ -327,8 +328,8 @@ def __init__(self, args, comet=None, proxy=None, al_iter=-1, data_path=None):
                 min_len=args.gflownet.min_seq_length,
                 max_len=args.gflownet.max_seq_length,
                 oracle=args.gflownet.func,
-                energy_weight=args.oracle.nupack_energy_reweighting,
-                nupack_target_motif=args.oracle.nupack_target_motif,
+                energy_weight=args.dataset.nupack_energy_reweighting,#oracle
+                nupack_target_motif=args.dataset.nupack_target_motif,#oracle
```

Comment:
Thanks for the fix but please remove comment at the end of the line for consistency :)